synapse/cortex.py

```diff
@@ -370,13 +367,27 @@ def __init__(self, dirn):
         self.setFeedFunc('syn.splice', self._addSynSplice)
         self.setFeedFunc('syn.ingest', self._addSynIngest)
 
-        # load any configured external layers
-        for path in self.conf.get('layers'):
-            logger.info('loading external layer: %r', path)
-            self.layers.append(s_layer.opendir.gen(path))
+        layersdir = pathlib.Path(dirn, 'layers')
```

Comment:
this is alot of code to kick around in `__init__` - can we move it into a private method?

Reply:
Yea, an _initCoreLayers() method wouldn't go amiss...